oak_utils/src/lib.rs

```diff
@@ -163,6 +163,20 @@ impl prost_build::ServiceGenerator for OakServiceGenerator {
     }
 }
 
+/// Options for generating Protocol buffer Rust types.
+pub struct ProtoOptions {
+    /// Generates service code for inter-node communication. Default: **true**.
```

Comment:
nit: could we make this even more explicit, to allow for other options/variants that will come later?  e.g. "Generates Oak-specific service code for inter-node communication; generated code will depend on the `oak` SDK crate."